.travis.yml

```diff
@@ -1,38 +1,64 @@
-dist: trusty
+dist: bionic
 sudo: enabled
 language: python
 python:
-  - "2.7"
-  - "3.5"
+  # - "3.5"
```

Comment:
We've been generally trying to keep Python 3.5 support for older Debian.  So I'd suggest re-enabling this.